src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj

```diff
@@ -19,14 +19,18 @@
             executables when it builds.  NuGet just happens to be a convenient way to share build artifacts.
         -->
         <IsPackable>true</IsPackable> 
+        <!-- Our outer Pack task (defined in PackAllRids.targets) invokes Pack passing in a PackRuntimeIdentifier to produce one package per RID; from that we can set everything else. -->
+        <RuntimeIdentifier Condition="'$(PackRuntimeIdentifier)' != '' and '$(PackRuntimeIdentifier)' != 'neutral'">$(PackRuntimeIdentifier)</RuntimeIdentifier>
+        <PackageId>$(AssemblyName).$(PackRuntimeIdentifier)</PackageId>
```

Comment:
Seems this needs to be conditioned on `'PackRuntimeIdentifier' != ''` to avoid the error

Reply:
The "actual" problem is we were creating a NuGet package for Microsoft.VisualStudio.LanguageServices.DevKit, which was calling into this project to the package name without a runtime identifier. Except such a package reference is unnecessary since these packages aren't meant to be consumed normally in the first place. It's also silly since the DevKit NuGet package wasn't being consumed either. So I'm just removing that setting in the DevKit project.